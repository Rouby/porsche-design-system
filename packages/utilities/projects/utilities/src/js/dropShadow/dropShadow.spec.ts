import * as fromDropShadow from './';

it('should provide all exports', () => {
  expect(Object.keys(fromDropShadow).length).toBe(3);
});

it.each<keyof typeof fromDropShadow>(Object.keys(fromDropShadow) as (keyof typeof fromDropShadow)[])(
  'should contain correct values for %s',
  (item) => {
    expect(fromDropShadow[item]).toMatchSnapshot();
  }
);
