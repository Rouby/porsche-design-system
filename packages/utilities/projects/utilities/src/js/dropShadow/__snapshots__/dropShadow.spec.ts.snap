// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should contain correct values for dropShadowHighStyle 1`] = `
{
  "filter": "drop-shadow(0px 8px 40px rgba(0, 0, 0, 0.16))",
}
`;

exports[`should contain correct values for dropShadowLowStyle 1`] = `
{
  "filter": "drop-shadow(0px 3px 8px rgba(0, 0, 0, 0.16))",
}
`;

exports[`should contain correct values for dropShadowMediumStyle 1`] = `
{
  "filter": "drop-shadow(0px 4px 16px rgba(0, 0, 0, 0.16))",
}
`;
